styles/prism-launcher/catppuccin.user.css

```diff
@@ -0,0 +1,136 @@
+==/UserStyle== */
+
+@-moz-document domain('prismlauncher.org') {
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+  :root[data-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    --accent: @base;
+    --accent-light: @base;
+    --background: @mantle;
+    --code: @base;
+    --code-fg: @text;
+    --danger: @peach;
+    --danger-dark: @peach;
+    --danger-light: @peach;
+    --disabled: @overlay0;
+    --grad-primary: @red;
+    --info: @crust;
+    --info-dark: @crust;
+    --info-light: @crust;
+    --inline-code: @base;
+    --inline-code-fg: @text;
+    --link: @accent-color;
+    --link-dark: @accent-color;
+    --link-light: @accent-color;
+    --primary: @red;
+    --primary-dark: @red;
+    --primary-light: @red;
+    --primary-text: @text;
+    --primary-text-muted: @subtext0;
+    --success: @green;
+    --success-dark: @green;
+    --success-light: @green;
+    --table-border: @surface1;
+    --warn: @yellow;
+    --warn-dark: @yellow;
+    --warn-light: @yellow;
+
+    .token.number {
+      color: @peach;
+    }
+
```

Comment:
```suggestion
    --warn: @yellow;
    --warn-dark: @yellow;
    --warn-light: @yellow;

    .button {
      color: @crust;
    }
    .notification.type-info {
      &,
      & summary,
      & a {
        color: @crust !important;
      }
    }

    .token.number {
      color: @peach;
    }

```
theme text in buttons to crust (white on accent isn't very readable)
before:
![image](https://github.com/user-attachments/assets/7aff0b96-1f31-4a05-940d-ccaac55b81d2)
after:
![image](https://github.com/user-attachments/assets/6eccf64b-e2e6-4dda-a4bd-848df53ceb6e)
and theme the text inside info boxes (links were invisible)
before:
![image](https://github.com/user-attachments/assets/0d216c64-4ff1-433f-871c-757fe6bae298)
after:
![image](https://github.com/user-attachments/assets/44bb43b6-0cf6-45c8-bbd9-fee35f57bfb5)

Reply:
Changed link buttons to blue color, should be more legible now. Info boxes are fixed, because they were changed to `crust` color